docker-compose.yml

```diff
@@ -23,6 +23,33 @@ services:
       # doesn't mask/overlay the true vendor tree which is populated in
       # the Dockerfile
       - /go/src/github.com/18F/hello-ampd/src/vendor
-    ports:
+    # don't map host port to container port--API is only available inside containter
```

Comment:
To clarify, what we want is actually 2 containers both listening on their own port. In a local environment, they'll be technically on the same IP, so they'll need to be on different ports. The ports on your local environment, don't matter much since nothing is blocked by firewalls, etc – `8080`, `8xxx` is pretty common – anything above `1024` for various reasons.

On cloud.gov, they'll be on different IPs, different hosts, but always using `443` since that is the standard port for HTTPS.